site/en/docs/extensions/mv3/file_handling/index.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: "layouts/doc-post.njk"
+title: "File Handling"
```

Comment:
Could we make this more specific, maybe File Handling on ChromeOS or something? Or Registering a file handler perhaps?